service/src/main/java/uk/gov/hmcts/reform/fpl/model/robotics/Applicant.java

```diff
@@ -16,11 +16,11 @@
     private final String jobTitle;
     private final Address address;
 
-    @Pattern(regexp = "^\\+?\\d+$", message = "is invalid")
+    @Pattern(regexp = "^\\+?\\d*$", message = "is invalid")
```

Comment:
As discussed I would remove it, but happy to create separate story for it tomorro.w